src/components/02-molecules/01-card/card--resource.njk

```diff
@@ -0,0 +1,67 @@
+{% if not standAlone %}
+<ul class="cards">
+{% endif %}
+	<li class="card__wrapper">
+		<article class="card card--resource">
+			<header>
+				<h3 class="card__title" >
+					<a href="{{ href }}">{{ title | safe }}</a>
+				</h3>
+				{% if byline %}
+					<p class="card__byline">By {{ byline | safe }}</p>
+				{% endif %}
+			</header>
+			<aside class="card__aside">
+				<div class="card__meta">
+					<span class="card__format"><span class="screen-reader-text">resource format: </span>{% render '@svg', {svg:formatIcon}, true %}{{ format }}</span>
+					{% if publisher %}
+					{% render '@separator' %}
```

Comment:
It seems that the presence of white space around the separator is forcing NVDA to say "dot" for the separator. If you remove the white space before the period, NVDA will pause.

To test this, I moved line 18 and appended it to line 16 without any printing space between spans.

Reply:
Okay, will do.